manual/qmcpack_papers.bib

```diff
@@ -0,0 +1,1676 @@
+  doi = {10.1103/PhysRevB.93.075143},
+  url = {http://link.aps.org/doi/10.1103/PhysRevB.93.075143}
+}
+
+% (app/method) clay H-He mix forces
+@article{clay2016b,
+  title = {Benchmarking density functionals for hydrogen-helium mixtures with quantum Monte Carlo: Energetics, pressures, and forces},
+  author = {Clay, Raymond C. and Holzmann, Markus and Ceperley, David M. and Morales, Miguel A.},
+  journal = {Phys. Rev. B},
+  volume = {93},
+  issue = {3},
+  pages = {035121},
+  numpages = {12},
+  year = {2016},
+  month = {Jan},
+  publisher = {American Physical Society},
+  doi = {10.1103/PhysRevB.93.035121},
+  url = {http://link.aps.org/doi/10.1103/PhysRevB.93.035121}
+}
+
+% (method) zhao excited state optimization
+@article{zhao2016,
+author = {Zhao, Luning and Neuscamman, Eric},
+title = {An Efficient Variational Principle for the Direct Optimization of Excited States},
+journal = {Journal of Chemical Theory and Computation},
+volume = {12},
+number = {8},
+pages = {3436-3440},
+year = {2016},
+doi = {10.1021/acs.jctc.6b00508},
+note ={PMID: 27379468},
+URL = { http://dx.doi.org/10.1021/acs.jctc.6b00508 },
+eprint = { http://dx.doi.org/10.1021/acs.jctc.6b00508 }
+}
+
+% (dev) mathuriya parallel bspline evaluation (arxiv)
+@article{mathuriya2016,
+  author    = {Amrita Mathuriya and
+               Ye Luo and
+               Anouar Benali and
+               Luke Shulenburger and
+               Jeongnim Kim},
+  title     = {Optimization and parallelization of B-spline based orbital evaluations
+               in {QMC} on multi/many-core shared memory processors},
+  journal   = {CoRR},
+  volume    = {abs/1611.02665},
+  year      = {2016},
+  url       = {http://arxiv.org/abs/1611.02665},
+  timestamp = {Thu, 01 Dec 2016 19:32:08 +0100},
+  biburl    = {http://dblp.uni-trier.de/rec/bib/journals/corr/MathuriyaLBSK16},
+  bibsource = {dblp computer science bibliography, http://dblp.org}
+}
+
+
+
+
+
+
+%2017
+% (method) zhao blocked linear method (arxiv)
+@article{zhao2017,
+  title={A Blocked Linear Method for Optimizing Large Parameter Sets in Variational Monte Carlo},
+  author={Zhao, Luning and Neuscamman, Eric},
+  journal={arXiv preprint arXiv:1702.01481},
+  year={2017},
+  url={https://arxiv.org/abs/1702.01481},
+}
+
+
+
+
+%unclear to count as qmcpack or not
+% (app) chiesa force estimator (did this paper use qmcpack, molecu or other?)
+@article{chiesa2005,
+  title = {Accurate, Efficient, and Simple Forces Computed with Quantum Monte Carlo Methods},
+  author = {Chiesa, Simone and Ceperley, D. M. and Zhang, Shiwei},
+  journal = {Phys. Rev. Lett.},
+  volume = {94},
+  issue = {3},
+  pages = {036404},
+  numpages = {4},
+  year = {2005},
+  month = {Jan},
+  publisher = {American Physical Society},
+  doi = {10.1103/PhysRevLett.94.036404},
+  url = {http://link.aps.org/doi/10.1103/PhysRevLett.94.036404}
+}
+
+
+% (app) spherical jellium qwalk (used orbitals from SQD code, packaged/part of qmcpack)
+@article{bajdich2011,
```

Comment:
I really don't know if QMCPACK was used in these cases.  Any reason not to comment these out pending further investigation?  (Only necessary due to auto-include)

Reply:
According to the paper, QMCPACK was used for the orbitals, so I think this counts.

Reply:
Agreed.  SQD was used in bajdich2011.  

What about the other two (chiesa2005 and swingle2013)?

Reply:
I emailed Norm to check Swingle2013, but considering we gained lots of Renyi code in the same time period, I think the answer is yes. Chiesa2005 is presumably no.